src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('tbody');
+const person = body.querySelectorAll('tr');
+const head = document.querySelector('thead');
+
+head.addEventListener('click', (e) => {
+  const order = e.target;
+
+  if (order.textContent === 'Name') {
+    const y = [...person].map(x => x);
+    const res = y.sort((a, b) =>
```

Comment:
`sort` is a mutating method, so there is no need to create new variable for sorted array
```suggestion
    y.sort((a, b) =>
```